setup.py

```diff
@@ -341,15 +339,10 @@ def test_otio():
     },
 
     # Disallow 3.9.0 because of https://github.com/python/cpython/pull/22670
-    python_requires='>2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*, !=3.9.0',  # noqa: E501
+    python_requires='>3.6, !=3.9.0',  # noqa: E501
```

Comment:
```suggestion
    python_requires='>=3.7, !=3.9.0',  # noqa: E501
```

`>3.6` means that 3.6.1 would be accepted:
```python
>>> import packaging.specifiers
>>> spec = packaging.specifiers.SpecifierSet('>3.6')
>>> spec.contains('3.6.1')
True
```